cargo-casper/schema-inject/src/lib.rs

```diff
@@ -0,0 +1,9 @@
+use casper_sdk::{casper::ret, casper_executor_wasm_common::flags::ReturnFlags};
+
+const SCHEMA: &str = r#""{{__CARGO_CASPER_INJECT_SCHEMA_MARKER}}""#;
```

Comment:
As discussed offline I believe there's a faster way to inject schema by passing the contents through compile-time env vars